packages/notification/src/components/GenericNotification.spec.tsx

```diff
@@ -1,11 +1,15 @@
-import { cleanup, render } from "@testing-library/react"
+import { act, render } from "@testing-library/react"
 import { fireEvent, waitFor } from "@testing-library/dom"
-import * as React from "react"
+import React from "react"
 import * as ReactTestUtils from "react-dom/test-utils"
 import GenericNotification from "./GenericNotification"
 
+const tick = () => new Promise(res => setImmediate(res))
+const advanceTimersByTime = async time =>
```

Comment:
Taken from here - https://stackoverflow.com/questions/66391541/jest-advancetimersbytime-doesnt-work-when-i-try-to-test-my-retry-util-function